org.lflang.tests/src/org/lflang/tests/LFTest.java

```diff
@@ -69,15 +69,14 @@ public LFTest(Target target, Path srcFile, Path packageRoot) {
 
     /** Stream object for capturing standard output. */
     public OutputStream getStandardOutput() {
-        return out;
+        return compilationLog;
```

Comment:
I don't think we need these methods anymore? If we do, we should just replace `getStandardOutput` and `getStandardError` with a single `getOutputStream`.